_drafts/oryx/orchestration/app-builder.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Application builder
+description: Builder of the Oryx Application
+template: concept-topic-template
+---
+
+# Application builder
```

Comment:
i wonder if we should keep this doc. What about removing it and move the required pieces to other docs (like the plugin docs?)